Libraries/LLM/Models.swift

```diff
@@ -98,11 +112,16 @@ public struct ModelConfiguration {
 extension ModelConfiguration {
 
     public static let mistral7B4bit = ModelConfiguration(
-        id: "mlx-community/Mistral-7B-v0.1-hf-4bit-mlx")
+        id: "mlx-community/Mistral-7B-v0.1-hf-4bit-mlx",
+
+        // https://www.promptingguide.ai/models/mistral-7b
+        defaultPrompt: "describe the swift language"
+    )
 
     public static let codeLlama13b4bit = ModelConfiguration(
         id: "mlx-community/CodeLlama-13b-Instruct-hf-4bit-MLX",
-        overrideTokenizer: "PreTrainedTokenizer"
+        overrideTokenizer: "PreTrainedTokenizer",
+        defaultPrompt: "func sortArray(_ array: [Int]) -> String { <FILL_ME> }"
```

Comment:
I verified each of these models and added a defaultPrompt so I don't have to hunt around each time

Reply:
Beautiful, thanks!